.github/workflows/pub-score.yml

```diff
@@ -0,0 +1,29 @@
+name: Pub Score Checker
+on:
+  pull_request:
+    branches:
+    - "main"
+
+jobs:
+
+  package-analysis:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: axel-op/dart-package-analyzer@v3
+        id: analysis
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+      - name: Check scores
+        env:
+          TOTAL: ${{ steps.analysis.outputs.total }}
+          TOTAL_MAX: ${{ steps.analysis.outputs.total_max }}
+        run: |
+          echo Score is: $TOTAL
+          if (( $TOTAL < 100 ))
```

Comment:
Since the package is currently at 120, there's no reason the pub score should go down unless something careless was done. I think we should only allow if it's 120 or above. And once this gets to 130 don't allow anything less than that